back-end/schemas/schemas.sql

```diff
@@ -29,13 +29,16 @@ CREATE TABLE IF NOT EXISTS Role (
 -- -----------------------------------------------------
 CREATE TABLE IF NOT EXISTS User (
     user_id INT UNIQUE NOT NULL AUTO_INCREMENT,
-    username VARCHAR(255) NOT NULL,
+    username VARCHAR(255) UNIQUE NOT NULL,
     email VARCHAR(255) UNIQUE NOT NULL,
     password VARCHAR(60) NOT NULL,
     phone VARCHAR(255) UNIQUE NOT NULL,
     role_id INT NOT NULL,
-    PRIMARY KEY (user_id),
+    create_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
+    updata_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
change the name to created_at instead and also remove the null because there is already a default value so it will not be null anyway

change the name to updated_at instead and it is fine to remove the default value and keep it just null and only modify it with the current time if an update happen to the row